src/modules/date/date-string.transformations.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@nestjs/common';
+import { DATE_FORMATS } from '@ukef/constants';
+import { DateOnlyString } from '@ukef/helpers/date-only-string.type';
+import { DateString } from '@ukef/helpers/date-string.type';
+import { matches } from 'class-validator';
+
+@Injectable()
+export class DateStringTransformations {
+  /**
+   * Remove time part from DateTime string.
+   */
+  removeTime(dateTime: string): DateString {
+    if (!dateTime) {
+      throw new TypeError(`Cannot remove the time from ${dateTime}.`);
+    }
+    return dateTime.split('T')[0] as DateString;
+  }
+
+  /**
+   * Check if dateTime input is not null or empty before removing time.
+   */
+  removeTimeIfExists(dateTime: string): DateString {
+    return (dateTime ? this.removeTime(dateTime) : dateTime) as DateString;
+  }
+
+  dateOnlyStringToDateString(dateOnlyString: DateOnlyString): DateString {
```

Comment:
Function names above have shorter names, not sure if we want to make them more similar. Everything longer or everything shorter.

Reply:
I've changed the name but it's still a bit longer than the others. I think the longer name is needed because I'm validating that it really is a DateOnlyString as well as adding the time part - what do you think?

Reply:
LGTM - we could have similar test in other functions, or just rely on typescript checks everywhere, but we can do this later.